node/src/actors/peers_manager/actor.rs

```diff
@@ -34,6 +34,9 @@ impl Actor for PeersManager {
                 // Get handshake time_out
                 act.handshake_timeout = config.connections.handshake_timeout;
 
+                // Get peers
```

Comment:
I don't think this comment describes what the next line is doing, hehehe.

Reply:
:man_facepalming: